lib/project_types/script/layers/application/push_script.rb

```diff
@@ -14,24 +14,29 @@ def call(ctx:, force:, project:)
 
             extension_point = ExtensionPoints.get(type: script_project.extension_point_type)
             library_name = extension_point.libraries.for(script_project.language)&.package
+
             raise Infrastructure::Errors::LanguageLibraryForAPINotFoundError.new(
               language: script_project.language,
               api: script_project.extension_point_type
-            ) unless library_name
+            ) unless library_name || (script_project.language == "wasm")
```

Comment:
I spoke offline with Alex about this, but it feels like the check here is really that the given API has support for the chosen language. In that case, should this be updated to checking `extension_point.libraries.for(script_project.language)` instead?

Reply:
That's _essentially_ what the library_name variable is, but I can change it up a bit to make it clearer that we actually care about the library here, not the name.